charts/pega/LogsPersistence.md

```diff
@@ -0,0 +1,36 @@
+Garbage collection logs and heap dumps are written to the pod storage which is ephemeral. When a pod crashes, all the logs and heap dumps stored in the pod are lost. To persist them we can use the following options:
```

Comment:
The name of the file is LogsPersistence but this document also talks about heapdump collection. Can this be named accordingly?

Reply:
Renamed the file.

Reply:
why do this file & the other files exist in `charts/pega` folder instead of `docs`? These `md` files may go into the chart archive, I don't think that is required.